### PR TITLE
Initial Documentation Content

### LINKED ISSUE #204: Docs Core Concepts content
As a user I would like to have a docs Core Concepts page.

Core Concepts

- Blockchain Basics: Overview of blockchain principles as they apply to social applications.
- Frequency Architecture: Detailed explanation of the Frequency infrastructure and how it supports social dapps.
- Frequency REST APIs vs. Traditional Web2 APIs: Comparison to help web2 developers understand the differences and similarities.
- Interruptibility between frequency social apps 

Please see master outline here
https://docs.google.com/document/d/1ZU968cvrR8eXjwP0vGFacUO_SCY901fRUPAD5dKruNg/edit#heading=h.t0m9wx7fa5dk

### LINKED ISSUE #205: Docs Getting Started content
As a user I would like to have a docs Getting Started page.

Getting Started

- A high-level guide on how the social web demo using the rest apis
- Configuration: Guide on configuring services for the first-time setup.
- First Project: Step-by-step tutorial to create a simple social app project.
- Make a provider on via a local node or TestNet using the provider dashboard 

Please see master outline here
https://docs.google.com/document/d/1ZU968cvrR8eXjwP0vGFacUO_SCY901fRUPAD5dKruNg/edit#heading=h.t0m9wx7fa5dk

### LINKED ISSUE #210: Docs Next Steps API Reference content
As a user I would like to have a docs API Reference page.

API Reference: (maybe swagger embed for all in 1)

- Detailed API documentation for all services, including request/response formats, authentication methods, and error codes.

Please see master outline here
https://docs.google.com/document/d/1ZU968cvrR8eXjwP0vGFacUO_SCY901fRUPAD5dKruNg/edit#heading=h.t0m9wx7fa5dk

### LINKED ISSUE #225: Docs:  Embed OpenAPI into Account Service Section
Title: Embed OpenAPI into Account Service Section

Description:
Embed the OpenAPI specification for the Account Service into the documentation site. This should include the full OpenAPI definition and any relevant usage examples or explanations.

Tasks:

- [ ] Retrieve the OpenAPI definition for the Account Service.
- [ ] Embed the OpenAPI definition into the Account Service section of the documentation site.
- [ ] Add any necessary explanations or usage examples to accompany the OpenAPI definition.

Acceptance Criteria:

- The Account Service section contains the full OpenAPI definition.
- The documentation includes clear examples and explanations for using the Account Service API.

### LINKED ISSUE #226: Docs:  Embed OpenAPI into Content Publisher Service Section
Title: Embed OpenAPI into Content Publisher Service Section

Description:
Embed the OpenAPI specification for the Content Publisher Service into the documentation site. This should include the full OpenAPI definition and any relevant usage examples or explanations.

Tasks:

- [ ] Retrieve the OpenAPI definition for the Content Publisher Service.
- [ ] Embed the OpenAPI definition into the Content Publisher Service section of the documentation site.
- [ ] Add any necessary explanations or usage examples to accompany the OpenAPI definition.

Acceptance Criteria:

- The Content Publisher Service section contains the full OpenAPI definition.
- The documentation includes clear examples and explanations for using the Content Publisher Service API.

### LINKED ISSUE #227: Docs: Embed OpenAPI into Content Watcher Service Section
Title: Embed OpenAPI into Content Watcher Service Section

Description:
Embed the OpenAPI specification for the Content Watcher Service into the documentation site. This should include the full OpenAPI definition and any relevant usage examples or explanations.

Tasks:

- [ ] Retrieve the OpenAPI definition for the Content Watcher Service.
- [ ] Embed the OpenAPI definition into the Content Watcher Service section of the documentation site.
- [ ] Add any necessary explanations or usage examples to accompany the OpenAPI definition.

Acceptance Criteria:

- The Content Watcher Service section contains the full OpenAPI definition.
- The documentation includes clear examples and explanations for using the Content Watcher Service API.

### LINKED ISSUE #228: Docs: Embed OpenAPI into Graph Service Section
Title: Embed OpenAPI into Graph Service Section

Description:
Embed the OpenAPI specification for the Graph Service into the documentation site. This should include the full OpenAPI definition and any relevant usage examples or explanations.

Tasks:

- [ ] Retrieve the OpenAPI definition for the Graph Service.
- [ ] Embed the OpenAPI definition into the Graph Service section of the documentation site.
- [ ] Add any necessary explanations or usage examples to accompany the OpenAPI definition.

Acceptance Criteria:

- 
- The Graph Service section contains the full OpenAPI definition.
- The documentation includes clear examples and explanations for using the Graph Service API.

### PR DESCRIPTION
**Pull request by wilwade**
This is the first pile of the documentation content.

- Anything not for this PR is tagged with "Coming soon"
- Lots of design to come
- Currently pulls in the swagger via https://www.npmjs.com/package/openapi-to-md (could be changed and improved in the future)

# Testing

```
cd docs
mdbook serve
open http://localhost:3000
```

Closes Docs issues:
- Closes #205
- Closes #204
- Closes #210

Closes embed issues:
- Closes #225
- Closes #226
- Closes #227
- Closes #228

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,6 @@ flowchart LR;
         GS[Graph Service]
         CPS[Content Publishing Service]
         CWS[Content Watcher Service]
-        RS[Graph Reconnection Service]
```
Comment (wilwade) [inline README.md:43]: Removing reconnection service for now

File: docs/css/extended.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
+/* Button Links */
```
Comment (wilwade) [inline docs/css/extended.css:1]: Button Link support

File: services/account/ENVIRONMENT.md [hunk 3]
```diff
@@ -1,23 +1,24 @@
 # Environment Variables
 
+ℹ️ Feel free to adjust your environment variables to taste.
 This application recognizes the following environment variables:
 
-| Name                                      | Description                                                                                                                                                                       |            Range/Type            | Required? | Default |
```
Comment (wilwade) [inline services/account/ENVIRONMENT.md:5]: Formatted and sorted

File: services/content-publishing/ENVIRONMENT.md [hunk 4]
```diff
@@ -1,22 +1,23 @@
...
 | `API_PORT`                                | HTTP port that the application listens on                                                                                                                                                                                                                                                |           1025 - 65535           |           |            3000             |
-| `CACHE_KEY_PREFIX`                        | Prefix to use for Redis cache keys                                                                                                                                                                                                                                                       |              string              |           | content-publishing-service: |
```
Comment (wilwade) [inline services/content-publishing/ENVIRONMENT.md:8]: Formatted and sorted

File: services/graph/ENVIRONMENT.md [hunk 5]
```diff
@@ -1,14 +1,15 @@
 # Environment Variables
 
+ℹ️ Feel free to adjust your environment variables to taste.
 This application recognizes the following environment variables:
 
 | Name                                      | Description                                                                                                                                                                       |            Range/Type            | Required? |     Default      |
-| ----------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------: | :-------: | :--------------: | --- | --- | --- | --- |
```
Comment (wilwade) [inline services/graph/ENVIRONMENT.md:6]: Fixed a typo, formatted, and sorted.

**Review by JoeCap08055 — APPROVED** [review]

File: docs/src/GettingStarted/README.md [hunk 6]
```diff
@@ -1,7 +1,106 @@
...
+## Step 3: Configure and Run the Example
+
+### Clone the Example Repository
+
+```sh
+git clone https://github.com/ProjectLibertyLabs/social-app-template.git
+```
+
+```sh
+cd social-web-template
```
Comment (wesbiggs) [inline docs/src/GettingStarted/README.md:59]: Typo, should be `social-app-template`

**Review by wesbiggs — COMMENTED** [review]
Haven't been through all of it but here are some suggestions.

File: docs/src/README.md [hunk 7]
```diff
@@ -1,8 +1,82 @@
...
+# Making Decentrialized Social Easy
 
-# Quick Start Guide
-Sed vel nulla nisl. Donec risus libero, auctor quis eros vitae, auctor elementum leo. Proin sed velit vel ex imperdiet aliquet. Phasellus varius dapibus ex at aliquam. Suspendisse dapibus urna vel augue molestie, sit amet semper erat fermentum. Suspendisse potenti. Sed vehicula neque quis metus lobortis hendrerit. Sed tristique, turpis non varius viverra, elit diam posuere arcu, vitae blandit ligula ipsum non libero. Quisque non dolor vitae urna pretium interdum in ac massa. Integer sapien sem, dapibus et suscipit sed, feugiat volutpat nibh. Nullam neque nisl, tempor vel eros ac, sodales euismod quam.
+Get started using Decentralized Social as easy as deploying a web2 API.
```
[the remark this answers is not in the document]
Comment (wilwade) [reply docs/src/README.md:3]: Ah. Nice. Swapping to `Get started building on`

File: docs/src/README.md [hunk 8]
```diff
@@ -1,8 +1,82 @@
...
 
-# Benefits of Migrating to Web3
-Integer in gravida risus, et dictum risus. Nam nulla leo, fermentum ac ultrices sed, venenatis ut magna. Suspendisse potenti. Nulla vehicula aliquet justo, ac hendrerit nunc dictum vitae. In volutpat cursus sapien, non pulvinar mauris sodales nec. Morbi consectetur vestibulum lorem, in ultricies sem euismod ut. Morbi et augue sit amet justo elementum bibendum eu eu neque. Vivamus egestas felis et lobortis bibendum. Praesent augue tortor, gravida eget nulla ac, tempus posuere nulla. Etiam pharetra gravida justo, at sodales lacus rhoncus eu. Curabitur vitae sagittis est.
+## Build What You Want
+
+Gateway offers a suite of tools you can pick and choose to build the best for your users.
+
+<!-- Add more visual elements to the services -->
+
+- Identity
```
[the remark this answers is not in the document]
Comment (wilwade) [reply docs/src/README.md:11]: 💯 Updating to:
```
- Add decentralized authentication and onboarding workflows
- Connect your users with their universal social graph
- Read, write, and interact with social media content
- More coming...
```

File: docs/src/README.md [hunk 9]
```diff
@@ -1,8 +1,82 @@
...
+## Web2 API Simplicity with Decentralized Power
+
+- Build your applications faster
+- Own your infrastructure
+- OpenAPI/Swagger out of the box
+- Optimized Docker images
+
+## Basic Architecture
+
+Gateway provides easy way to interact with Frequency without requiring any direct chain interactions.
```
[the remark this answers is not in the document]
Comment (wilwade) [reply docs/src/README.md:25]: I was trying to balance "this makes it easy" (good to say) against "blockchain is hard" (not as good to say as it is only somewhat true).

Swapping to positive statement for now. Open to other ideas, but the list I think now works with having updated the list higher up.
```
Gateway provides a simple API to interact with the Frequency social layers of identity, graph, content, and more.
```

File: docs/src/README.md [hunk 10]
```diff
@@ -1,8 +1,82 @@
...
+### Content Publishing Service
+
+The Content Publishing Service is a service enabling the creation of new content related activity on Frequency.
+
+- Create posts to be broadcast publicly
+- Create replies to posts
+- Create reactions to posts
+- Create updates to existing content
+- Request deletion of content
+- Asset creation with [IPFS](https://ipfs.tech)
```
[the remark this answers is not in the document]
Comment (wilwade) [reply docs/src/README.md:64]:
```suggestion
- Media attachement storage with [IPFS](https://ipfs.tech)
```

File: docs/src/Fundamentals/README.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+Learn more about Delegation in [Frequency Documentation](https://docs.frequency.xyz/Delegation/).
+
+## Interoperability Between Apps
+
+Frequency enables seamless interaction and data sharing between different applications built on its platform.
+This interoperability is facilitated by:
+
+- **Standardized Protocols:** Frequency uses the Decentralized Social Networking Protocol (DSNP), an open web3 protocol that ensures compatibility between different applications.
+- **Common Data Structures:** By using standardized data structures for user profiles, messages, and other social interactions, Frequency ensures that data can be easily shared and interpreted across different applications.
+- **Interoperable APIs:** Frequency provides a set of REST APIs that allow developers to build applications capable of interacting with each other, ensuring a cohesive user experience across the ecosystem.
```
[the remark this answers is not in the document]
Comment (wilwade) [reply docs/src/Fundamentals/README.md:25]: I think that was originally thinking about SIWF, but removing the line for now.

**Pull request merged**